Dockerfile

```
@@ -1,15 +1,14 @@
 # The base image is expected to contain
 # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-ARG BASE_IMAGE=registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.18
 FROM ${BASE_IMAGE}
```

```suggestion
FROM ${BASE_IMAGE}
ARG OCP_VESION[=<default>?]
```

define the arg and set a default consistently with the decision for https://github.com/openshift/multiarch-tuning-operator/pull/571/files#r2045542977